README.md

```diff
@@ -1,32 +1,50 @@
-# WordPress Headless Framework (PREVIEW/ALPHA)
+# Headless WordPress Framework
+
+## Introduction
+
+WPEngine's Headless WordPress Framework provides a set of tools to make building front-end applications with WordPress as the headless CMS a pleasant experience for both developers and publishers. This framework consists of a WordPress plugin, a set of npm packages, and guides to get you started building headless WordPress sites in Next.js.
+
+_🚧 **Note:** This project is in the early stages of development_
 
-🚧 **Note:** This project is in the early stages of development, but it does contain useful functionality for headless WordPress sites like plugins and npm packages that assist in authentication and previews.
 
 ## Quick Start
 
-Eager to try out the Headless Framework? Here's how you can get started with our Preview example:
+Eager to try out the Headless Framework? Here's how you can get started:
 
 1. Create a WordPress site if you haven't already. We recommend using [Local](https://localwp.com/)!
-2. Download, upload, and activate the `wpe-headless` plugin in this repository. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
+2. Download, upload, and activate the `wpe-headless` plugin. [(Plugin Download)](https://wp-product-info.wpesvc.net/v1/plugins/wpe-headless?download)
 3. Install [WP GraphQL](https://wordpress.org/plugins/wp-graphql/) on the WordPress site if it's not already installed
-4. [Clone this repository](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) to a directory of your choice
-5. Navigate to `examples/preview` in the cloned repository
+4. Download our [Getting Started example project](https://github.com/wpengine/headless-framework/tree/canary/examples/getting-started) and navigate to it locally
 6. `cp .env.local.sample .env.local`
 7. Populate `NEXT_PUBLIC_WORDPRESS_URL` and `WPE_HEADLESS_SECRET` accordingly in `.env.local`
 8. `npm install && npm run dev`
```

Comment:
Perhaps make it more clear what `NEXT_PUBLIC_WORDPRESS_URL` should contain, and how to find the `WPE_HEADLESS_SECRET`:

```suggestion
6. `cp .env.local.sample .env.local`
7. Populate `NEXT_PUBLIC_WORDPRESS_URL` in `.env.local` with the full URL to your WordPress site, including the `http://` or `https://` prefix.
8. Populate the `WPE_HEADLESS_SECRET` in `.env.local` with the secret key found at Settings → Headless in your WordPress admin area.
9. `npm install && npm run dev`
```

Reply:
Good callouts, thanks!